SEED-platform/seed
# Fix program setup's new program button action

#### Any background context you want to provide?
The "New Program" button on the program insights config page would save invalid programs to the db on each click. 

#### What's this PR do?
New programs are only created when they have passed validation via the 'Save Changes' button

#### How should this be manually tested?
Go to Program Insights config modal and click "New Program" multiple times. Nothing should happen after the first click.

#### What are the relevant tickets?
#4321

#### Screenshots (if appropriate)
